pkg/certificate/request.go

```diff
@@ -47,7 +47,16 @@ type Request struct {
 	FetchPrivateKey bool
 	/*	Thumbprint is here because *Request is used in RetrieveCertificate().
 		Code should be refactored so that RetrieveCertificate() uses some abstract search object, instead of *Request{PickupID} */
-	Thumbprint       string
+	Thumbprint string
+	// Timeout we have multiple purposes for timeout (bad practice, we need to correct this in the future)
```

Comment:
If this is something we should not encourage. Why adding another meaning and purpose to this attribute instead of creating a new one?

Reply:
Ideally we need to standardize this, as there a many different usage, but is not the purpose of this MR hence I mentioned what we already:
> bad practice, we need to correct this in the future

Without any standardization **as user**, I'd expect the request attribute `Timeout`, at very least serve, the purpose, that allows me to increase the timeout for enrolling certificate, which it is what I'm proposing we do for TLSPDC side of things. (As of now, that attribute has no use in TLPSDC for certificate enrollment).

Reply:
Done by latest commit